src/rateEngine/RateComponent.ts

```diff
@@ -49,6 +50,26 @@ class RateComponent {
   rateElementClassification() {
     return this._billingDeterminants.rateElementClassification;
   }
+
+  formatCharge(): string {
+    switch (this._classification) {
+      case RateElementClassification.ENERGY: {
+        return _mean(this.charge).toFixed(5);
```

Comment:
`charge` gets expanded from a number into an array when the instance is constructed which seems a little off possibly.

I think instead, `charge` should be a number and there should be something like a `chargePerMonth` method. The result would be just passing `this.charge.toFixed` instead of taking the mean.

That change is out of scope for at least the initial changes in this PR, which is why I haven't touched it.